### PR TITLE
Settings menu enhancements

### PR DESCRIPTION
**Pull request by LinuxLlama**
Better categorization, strings, and looks. Makes the settings menu more user-friendly and less cluttered.

### CODE REVIEW

File: app/src/main/res/xml/launcher_dock_preferences.xml [hunk 1]
```diff
@@ -1,89 +1,105 @@
...
+                android:defaultValue="true"
+                android:persistent="true" />
 
-        <SwitchPreference
-            android:key="pref_hotseatShowArrow"
-            android:title="@string/hotseat_show_arrow"
-            android:defaultValue="true"
-            android:persistent="true" />
+            <SwitchPreference
+                android:key="pref_isHotseatTransparent"
```
Comment (divadsn) [inline app/src/main/res/xml/launcher_dock_preferences.xml:92]: Doesn't it require to also change the preference key in the code?

**Review by divadsn — CHANGES_REQUESTED** [review]

**Comment by divadsn** [comment]
Waiting for @deletescape to confirm

File: app/src/main/res/xml/launcher_dock_preferences.xml (hunk 1 above)
Comment (nyancrimew) [reply app/src/main/res/xml/launcher_dock_preferences.xml:92]: He didn't change preference keys, he just moved some preferences around, this one used to be located a bit further up in the file.

Comment (LinuxLlama) [reply app/src/main/res/xml/launcher_dock_preferences.xml:92]: Yeah, it doesn't need to be changed. It's already added up a bit like what Till said,

Comment (divadsn) [reply app/src/main/res/xml/launcher_dock_preferences.xml:92]: Oh I see, stupid git diff.

**Pull request merged**